image/share/Taskfile

```diff
@@ -0,0 +1,3 @@
+default: KMER=121 PARTITIONS=4
+fast: KMER=63 PARTITIONS=128
+faster: KMER=31 PARTITIONS=256
```

Comment:
@michaelbarton I think using minimal base image is a good idea but could you add/reimplement fernandos tasks. For example the cutoffs as defined here: https://github.com/bioboxes/velour/blob/velour/Taskfile are completely missing